dts/arm64/renesas/rcar_gen3_ca57.dtsi

```diff
@@ -29,7 +29,7 @@
 	};
 
 	gic: interrupt-controller@f1010000 {
-		compatible = "arm,gic-400", "arm,gic" ;
+		compatible = "arm,gic-400", "arm,gic-v2", "arm,gic" ;
```

Comment:
in `arm,gic` still needed here?

Reply:
as far as I know, yes
we obtain GIC register properties using the arm,gic compatible node:
```
#define GIC_DIST_BASE	DT_REG_ADDR_BY_IDX(DT_INST(0, arm_gic), 0)
#define GIC_CPU_BASE	DT_REG_ADDR_BY_IDX(DT_INST(0, arm_gic), 1)
```
The `arm,gic-v[1-3]` compatible strings are used specifically to specify the GIC version.

Reply:
I see, ok thanks.